packages/storefront/tests/a11y/specs/accessibility.a11y.ts

```diff
@@ -40,10 +42,14 @@ const gotoUrl = async (page: Page, url: string): Promise<void> => {
 };
 
 const enableDarkMode = async (page: Page): Promise<void> => {
-  const themeBtn = await page.$('.cycle-platform-theme');
-  await themeBtn.click();
+  if (await page.locator('.cycle-platform-theme').count()) {
```

Comment:
Could extract `page.locator('.cycle-platform-theme')` into const to avoid calling it two times